### ISSUE TITLE
Preservation Catalog Jun 2018 update

### ISSUE THREAD

**Issue by ptatian** [NeighborhoodInfoDC/HUD#99]
_Note: If you have not previously done so, [clone the HUD repo](https://help.github.com/desktop/guides/contributing-to-projects/cloning-a-repository-from-github-to-github-desktop/) to **D:\DCData\Libraries** before proceeding._
- [ ] [Create new branch](https://help.github.com/desktop/guides/contributing-to-projects/creating-a-branch-for-your-work/) **update-#**_n_ in GitHub Desktop for work on this issue (where _n_=issue number). 
- [ ] [Update Section 8 multifamily data](https://urbanorg.app.box.com/notes/44777865557) (if new update available)
- [ ] [Update MFIS insured mortgages data](https://urbanorg.app.box.com/notes/46811827989) (if new update available)
- [ ] [Request review and finalize programs](https://urbanorg.app.box.com/notes/295752223069)

**Comment by ptatian** [comment]
Hi @Woliver01 and @mcohenui. I've created a new GitHub issue in the HUD repo for updating the section 8 and MFIS data for this month's Preservation Network meeting. As I mentioned, I've tried to streamline the steps so we now just have one issue for all the monthly HUD updates. I hope that will simplify things. 

I'll be setting up a couple of other issues for the DHCD data and the Catalog updates. But, for now, Will can start on this one. 

Let me know if you have any questions. Thanks!

Charge code: 101592-0001-001-00001

cc: @rpitingolo
